app/controllers/api/categories/items_controller.rb

```diff
@@ -2,8 +2,7 @@ module Api
   module Categories
     class ItemsController < ApiController
       def index
-        # chosen_item = Item.where(category: params[:category_id]) works aswell
-        chosen_items = Item.all.select { |i| i.category.downcase == params[:category_id] }
+        chosen_items = Item.where(category: params[:category_id])
```

Comment:
You actually correctly used the pattern for comparing Downcase values of categories in `app/controllers/api/categories_controller.rb`, i.e. the `where('lower.... = ?'...)`
You should use it here as well.